Build Error on macOS: default initialization of an object of const type 'const VehicleParams' without a user-provided default constructor.

Compiler version: `clang-800.0.42.1`.
Build error message:
```bash
/Users/JMSuen/Repositories/AirSim/AirLib/src/controllers/MavLinkDroneController.cpp:890:36: error: default initialization of an object of const type
      'const msr::airlib::VehicleParams' without a user-provided default constructor
        static const VehicleParams vehicle_params_;
                                   ^
                                                  {}
1 warning and 1 error generated.
make[3]: *** [AirLib/CMakeFiles/AirLib.dir/Users/JMSuen/Repositories/AirSim/AirLib/src/controllers/MavLinkDroneController.cpp.o] Error 1
make[2]: *** [AirLib/CMakeFiles/AirLib.dir/all] Error 2
make[1]: *** [AirLib/CMakeFiles/AirLib.dir/rule] Error 2
make: *** [AirLib] Error 2
```

According to [this SO page](http://stackoverflow.com/questions/7411515/why-does-c-require-a-user-provided-default-constructor-to-default-construct-a), `struct VehicleParams` should be declared as non-POD.

--- a/AirLib/include/controllers/DroneCommon.hpp
+++ b/AirLib/include/controllers/DroneCommon.hpp
@@ -40,6 +40,7 @@ struct YawMode {
 
 //properties of vehicle
 struct VehicleParams {
+    VehicleParams(){}; 
     //what is the breaking distance for given velocity?
     //currently we support simple linear relationship
     float vel_to_breaking_dist = 0.5f;   //ideally this should be 2X for very high speed but for testing we are keeping it 0.5

